addons/l10n_fr_pos_cert/i18n/fr.po

```diff
@@ -164,26 +178,22 @@ msgid "Date to which the values are computed"
 msgstr "Date jusqu'à laquelle les valeurs sont calculées"
 
 #. module: l10n_fr_pos_cert
-#. openerp-web
+#. odoo-javascript
 #: code:addons/l10n_fr_pos_cert/static/src/js/pos.js:0
 #, python-format
 msgid "Deleting of orders is not allowed."
 msgstr "La suppression de commandes n'est pas autorisée."
 
 #. module: l10n_fr_pos_cert
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_account_bank_statement__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_account_bank_statement_line__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_account_fiscal_position__display_name
 #: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_account_sale_closing__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_pos_config__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_pos_order__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_pos_order_line__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_pos_session__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_report_l10n_fr_pos_cert_report_pos_hash_integrity__display_name
-#: model:ir.model.fields,field_description:l10n_fr_pos_cert.field_res_company__display_name
 msgid "Display Name"
 msgstr "Nom affiché"
 
+#. module: l10n_fr_pos_cert
+#: model_terms:ir.ui.view,arch_db:l10n_fr_pos_cert.report_pos_hash_integrity
+msgid "Données corrompues sur la commande du point de vente:"
```

Comment:
is it normal that it is already in french ? 🤔

Reply:
You can use `t-translation="off"` in the report to disable translations when not needed

Reply:
Probably there initially because it was made for FR but wouldn't hurt to have an English translation
"Corrupted data on the PoS order:"

BTW thanks @lse-odoo for the other translations 👍